### PR TITLE
Remove dummy controllers

### PR DESCRIPTION
**Pull request by DonJayamanne**
Reverting due to https://github.com/microsoft/vscode/issues/124535
(tracked here #5927)

### CODE REVIEW

File: src/client/datascience/notebook/notebookControllerManager.ts [hunk 1]
```diff
@@ -195,7 +182,7 @@ export class NotebookControllerManager implements INotebookControllerManager, IE
                     traceInfo('Find preferred kernel cancelled');
                     return;
                 }
-                await this.createOrDeletePlaceholderPythonKernel();
```
Comment (rchiodo) [inline src/client/datascience/notebook/notebookControllerManager.ts:198]: If I understand the issue correctly the id that the dummy kernel has needs to be updated in order for it to work after you reload VS code?

Perhaps this isn't true? Can we just use the same id for the dummy kernel forever? Save it in user memento or something.

Comment (DonJayamanne) [reply src/client/datascience/notebook/notebookControllerManager.ts:198]: >  use the same id for the dummy kernel forever? Save it in user memento or something.

Yes agreed, we could (i didnt want to add that code here), if i did go down that path, I'd be  adding the solution for pre-loads where we have a dummy solution.

Comment (DonJayamanne) [reply src/client/datascience/notebook/notebookControllerManager.ts:198]: I.e. i'd like to fix this problem once we have a better understanding of the problem & solution. Right now it seems like i'm going down the path of fixing (coming up with a solution) as we implement something.

Comment (rchiodo) [reply src/client/datascience/notebook/notebookControllerManager.ts:198]: So you're backing out the idea of a dummy kernel entirely, and waiting to do this until we decide on how to handle preload kernels?

**Review by rchiodo — APPROVED** [review]

File: src/client/datascience/notebook/notebookControllerManager.ts (hunk 1 above)
Comment (IanMatthewHuff) [reply src/client/datascience/notebook/notebookControllerManager.ts:198]: Current implementation feels super reactionary to me, so I'm super pro backing off and approaching this with more of a plan.

**Review by IanMatthewHuff — APPROVED** [review]

**Pull request merged**